projects/storefrontstyles/scss/components/cart/_cart-coupon.scss

```diff
@@ -3,12 +3,8 @@
 %cx-cart-coupon {
   padding-inline-end: 0;
   padding-inline-start: 3rem;
-  padding-top: 2rem;
-  padding-bottom: 2rem;
 
   @include media-breakpoint-down(md) {
-    padding-top: 0;
```

Comment:
put padding-top: 2rem to align with cart-totals unless we're stacking it in tablet

Reply:
but to reset it back to 0 in the `sm` media query